Packages/Tracking/Core/Runtime/Scripts/XRHandsSubsystem/Scripts/LeapHandsSubsystem.cs

```diff
@@ -0,0 +1,170 @@
+using Unity.Collections;
+using UnityEngine;
+using UnityEngine.XR.Hands;
+using UnityEngine.SubsystemsImplementation;
+using UnityEngine.XR.Hands.ProviderImplementation;
+
+using Leap;
+using Leap.Unity;
+using System;
+using static UnityEngine.XR.Hands.XRHandSubsystem;
+using UnityEngine.XR.Hands.OpenXR;
+
+namespace Leap.Unity
+{
+    class LeapXRHandProvider : XRHandSubsystemProvider
+    {
+        LeapProvider provider;
+
+        public override void Destroy()
+        {
+        }
+
+        public override void GetHandLayout(NativeArray<bool> handJointsInLayout)
+        {
+        }
+
+        public override void Start()
+        {
+            provider = GameObject.FindObjectOfType<XRLeapProviderManager>();
```

Comment:
this should use Hands.Provider to find _any_ most suitable provider, as the Manager is not specifically necessary